HARK/distribution.py

```diff
@@ -1404,3 +1664,48 @@ def sample(s):
         array_sample = np.frompyfunc(sample, 1, 1)
 
         return array_sample(state)
+
+
+def ExpectedValue(func=None, dist=None, args=(), labels=False):
```

Comment:
I am -1 on capitalizing this function.
I know that's a minority opinion, and I expect to be overruled.
But I want to note that I think this is short-sighted.
While it may make it marginally easier to _write_ the code, for those who are not accustomed to Python conventions, it will make it more difficult to _read_ the code.
When the project is successful, the code will be read much more often than it is written.
And this choice will ultimately be costly and determined to be unwise.

Reply:
At the very least, if this remains in CamelCase, then I think the departure from PEP8 and its rationale should be in the documentation.
That way when people look it up in the autodocs there's something to guide them in their confusion about why it isn't a class.

Reply:
I see you are also still using a tuple for args here.
This is quite confusing given the way 'expected_value' methods on Distribution objects are designed differently.
I expect (no pun intended) you would prefer a parallel construction.

What happens if you use `*args` instead? You mentioned a complication with this on an earlier version of the code.